server/proxy/httpproxy/director.go

```diff
@@ -33,7 +33,7 @@ func init() {
 	rand.Seed(time.Now().UnixNano())
 }
 
-func newDirector(lg *zap.Logger, urlsFunc GetProxyURLs, failureWait time.Duration, refreshInterval time.Duration) *director {
+func newDirector(lg *zap.Logger, urlsFunc GetProxyURLs, failureWait time.Duration, refreshInterval time.Duration, stop <-chan struct{}, donec chan<- struct{}) *director {
```

Comment:
`stop` -> `stopc` so as to be consistent with `donc`?

Reply:
Agreed. With the suggestion of wrapping `stopc` and `donec` in director, this consistency looks much nicer.